staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -74,8 +74,12 @@ func NewCmdVersion(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *co
 		Long:    i18n.T("Print the client and server version information for the current context."),
 		Example: versionExample,
 		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) > 0 {
```

Comment:
This block already runs in Validate func?

Reply:
Sorry, I forgot removing it. Fixed